### DISCUSSION TITLE
Curso Open

### DISCUSSION THREAD

**Discussion by OjoseSalazar** [rramosp/ai4eng.v1#254] [UDEA Students]
Hace unos dias un monitor me escribio que no se habian podido subir mis notas, debido a que aparezco en el curso abierto en la plataforma de calificaciones, como debo proceder para solucionar eso?

**Comment by imEag** [comment]
Debes matricularte en el curso correcto [aquí](https://m5knaekxo6.execute-api.us-west-2.amazonaws.com/dev-v0001/rlxmooc/web/request_invitation/ai4eng.v1.udea.r2/2023.1) y luego migraremos tus notas.

**Reply by OjoseSalazar** [reply]
Ya me matricule en el curso

**Reply by imEag** [reply]
vale Jose, nos encargaremos de migrar tus notas al curso correcto

**Reply by imEag** [reply]
Ya están listas tus notas en el curso
